README.md

```diff
@@ -104,18 +104,35 @@ DEPRECATION_CONFIG:
         example: "2011-12-25T23:59:59"
       matchPattern:
         type: string
-        description: "A regex string pattern to match against requests"
+        description: "A regex string pattern to match against requests. If you use ".*/data$" this is for use with the 'pageTypes' below to deprecate data endpoints"
+      message:
+        type: string
+        description: "A message to return in the case of data endpoints"
+      pageTypes:
+        type: string
+        description: "Comma separated list of page types, in the singular"
+        example: "bulletin,article"
 ```
 
 For example:
 
 ```json
 [
+  // Matches by the match pattern
   {
     "deprecationDate": "2011-11-30T23:59:59",
     "deprecationLink": "https://developer.ons.gov.uk/retirement/",
     "sunsetDate": "2011-12-25T23:59:59",
     "matchPattern": "^/timeseriestool/data$"
+  },
+  /// Matches on the data endpoint
+  {
+    "deprecationDate": "2011-11-30T23:59:59",
+    "deprecationLink": "https://developer.ons.gov.uk/retirement/",
+    "sunsetDate": "2011-12-25T23:59:59",
```

Comment:
When I was testing, it wasn't clear if this datetime was local or UTC. I have determined it is local, but maybe somewhere in this README it should clarify this.